src/CCFieldValidator.js

```diff
@@ -2,13 +2,23 @@ import valid from "card-validator";
 import pick from "lodash.pick";
 import values from "lodash.values";
 import every from "lodash.every";
+import { CARDS_OVERRIDES } from "./cardsOverrides";
+
+const MAESTR0_BASE_LENGTH = 12;
+CARDS_OVERRIDES.forEach(card => valid.creditCardType.addCard(card));
 
 const toStatus = validation => {
   return validation.isValid ? "valid" :
          validation.isPotentiallyValid ? "incomplete" :
          "invalid";
 };
 
+const hasProperLength = cardNumber => cardNumber.split(" ").join("").length >= MAESTR0_BASE_LENGTH;
+const maestroCardStatus = (numberValidation, cardNumber) =>
+  hasProperLength(cardNumber)
+  && numberValidation.card
+  && numberValidation.card.type === "maestro" && "valid";
+
```

Comment:
Since we are mantaining or own fork, let's keep this a bit cleaner:

Rename `cardsOverrides.js` to `constants.js` and move `MAESTRO_BASE_LENGTH` there.

Create a `utils.js` file and move `toStatus`, `hasProperLength` and `maestroCardStatus` there.

Move `"maestro"` and `"valid"` to constants.js as `MAESTRO_TYPE='maestro'` and `VALID='valid'`

Make `hasProperLength` and `cardStatus` a bit more universal:

```javascript

// utils.js

const hasProperLength = (cardNumber, length) => cardNumber.split(" ").join("").length >= length;

const cardStatus = (numberValidation, cardNumber, type, length) =>
  hasProperLength(cardNumber, length)
  && numberValidation.card
  && numberValidation.card.type === type && VALID;

export const maestroCardStatus = (numberValidation, cardNumber) =>  cardStatus(numberValidation, cardNumber, MAESTRO_TYPE, MAESTRO_BASE_LENGTH)
```

So if we have another problem like this, it's clearer and easier for anyone to come and fix it with this solution of yours.

Reply:
Done!.